deepmd/dpmodel/utils/network.py

```diff
@@ -581,6 +581,7 @@ def __init__(
             resnet_dt: bool = False,
             precision: str = DEFAULT_PRECISION,
             seed: Optional[Union[int, List[int]]] = None,
+            bias: bool = True,
```

Comment:
`bias` should be added to the docstring.

Reply:
should we name this `use_tebd_bias`

Reply:
> should we name this `use_tebd_bias`

It's in `make_embedding_network`, which is also used by other layers. So `bias` may be more suitable.